trio/_threads.py

```diff
@@ -1,3 +1,5 @@
+# coding: utf-8
+
```

Comment:
these headers should not be needed, right?

in the few files where these appears, git blame is always referencing your PR's.  Over-eager editor setting?

Reply:
@belm0 Not needed indeed, feel free to remove those lines